zebra-chain/src/orchard/sinsemilla.rs

```diff
@@ -201,6 +201,7 @@ pub fn sinsemilla_short_commit(
 mod tests {
```

Comment:
In general, I'd like to have tests in a separate file. I think we have a follow up ticket for test modules already, can you make sure this module is listed?